src/pages/ReimbursementAccount/ConnectBankAccount/ConnectBankAccount.tsx

```diff
@@ -39,7 +39,7 @@ function ConnectBankAccount({reimbursementAccount, onBackButtonPress, account, p
     const styles = useThemeStyles();
     const {translate} = useLocalize();
 
-    const handleNavigateToConciergeChat = () => Report.navigateToConciergeChat(false, true);
+    const handleNavigateToConciergeChat = () => Report.navigateToConciergeChat(true);
```

Comment:
Fixing the param conflict

Reply:
The param to dismiss the modal was added in bank account refactor.

Still works fine
https://github.com/Expensify/App/assets/50919443/c82806b6-69cf-4322-b5e1-59e977e99003